samples/lib/thrift/hello_server/src/main.cpp

```diff
@@ -27,44 +27,40 @@ using namespace ::apache::thrift::server;
 #ifdef __ZEPHYR__
 void main(void)
 #else
-int main(int argc, char **argv)
+int main(int argc, char** argv)
 #endif
 {
-    ::std::string my_addr;
+  ::std::string my_addr;
 
 #ifdef __ZEPHYR__
-    my_addr = CONFIG_NET_CONFIG_MY_IPV4_ADDR;
+  my_addr = CONFIG_NET_CONFIG_MY_IPV4_ADDR;
 #else
-    if (argc < 2)
-    {
-        printf("usage: %s <ip>\n", argv[0]);
-        return EXIT_FAILURE;
-    }
+  if (argc < 2) {
+    printf("usage: %s <ip>\n", argv[0]);
+    return EXIT_FAILURE;
+  }
 
-    my_addr = std::string(argv[1]);
+  my_addr = std::string(argv[1]);
 #endif
 
-    int port = 4242;
-    ::std::shared_ptr<HelloHandler> handler(new HelloHandler());
-    ::std::shared_ptr<TProcessor> processor(new HelloProcessor(handler));
-    ::std::shared_ptr<TServerTransport> serverTransport(new TServerSocket(my_addr, port));
-    ::std::shared_ptr<TTransportFactory> transportFactory(new TBufferedTransportFactory());
-    ::std::shared_ptr<TProtocolFactory> protocolFactory(new TBinaryProtocolFactory());
-    TSimpleServer server(processor, serverTransport, transportFactory, protocolFactory);
+  int port = 4242;
+  ::std::shared_ptr<HelloHandler> handler(new HelloHandler());
```

Comment:
It looks as though it's much more common to use `std::` than `::std::` upstream. Let's stick with `std::` here.
```shell
cfriedt@cfriedt-mbp .upstream % grep -R "::std" * | wc -l
      65
cfriedt@cfriedt-mbp .upstream % grep -R "std" * | grep -v "::std" | wc -l
    8037
```

Looking much better already!